back/Dockerfile

```diff
@@ -74,10 +74,11 @@ RUN apt-get update \
     netcat=1.10-46 \
     && rm -rf /var/lib/apt/lists/*
 
-COPY ./entrypoint.dev.sh .
-RUN chmod +x /src/entrypoint.dev.sh
+COPY ./entrypoint.dev.sh /usr/local/bin/
```

Comment:
I don't understand why you didn't copy the entrypoint.dev.sh in the same folder than the entrypoint.prod.sh

Reply:
because I am stupid, I forgot to change that... :smile: